geode-redis/README.md

```diff
@@ -128,210 +128,61 @@ not connected>
-|                    	|                                                    	| ZCARD                         	|
-|                    	|                                                    	| ZCOUNT                        	|
-|                    	|                                                    	| ZINCRBY                       	|
-|                    	|                                                    	| ZINTERSTORE                   	|
-|                    	|                                                    	| ZLEXCOUNT                     	|
-|                    	|                                                    	| ZPOPMAX                       	|
-|                    	|                                                    	| ZPOPMIN                       	|
-|                    	|                                                    	| ZRANGE                        	|
-|                    	|                                                    	| ZRANGEBYLEX                   	|
-|                    	|                                                    	| ZRANGEBYSCORE                 	|
-|                    	|                                                    	| ZRANK                         	|
-|                    	|                                                    	| ZREM                          	|
-|                    	|                                                    	| ZREMRANGEBYLEX                	|
-|                    	|                                                    	| ZREMRANGEBYRANK               	|
-|                    	|                                                    	| ZREMRANGEBYSCORE              	|
-|                    	|                                                    	| ZREVRANGE                     	|
-|                    	|                                                    	| ZREVRANGEBYSCORE              	|
-|                    	|                                                    	| ZREVRANK                      	|
-|                    	|                                                    	| ZSCAN                         	|
-|                    	|                                                    	| ZSCORE                        	|
-|                    	|                                                    	| ZUNIONSTORE                   	|                   	|
-
-**NOTES:**
-
-[1] - UNLINK is implemented as a synonym to DEL and does not unlink asynchronously.
-[2] - SLOWLOG is implemented as a NoOp.
+The Geode APIs compatible with Redis currently implement a subset of the full Redis command set.
+
+#### <a name="supported-redis-commands"></a> Supported Redis Commands [Return to top](#introduction)
+- APPEND 
+- AUTH
+- DECR
+- DECRBY
+- DEL
+- EXISTS
+- EXPIRE
+- EXPIREAT
+- GET
+- GETRANGE
+- HDEL
+- HEXISTS
+- HGET
+- HGETALL
+- HINCRBY
+- HINCRBYFLOAT
+- HLEN
+- HMGET
+- HMSET
+- HSCAN
+- HSET
+- HSETNX
+- HSTRLEN
+- HVALS
+- HKEYS
+- INCR
+- INCRBY
+- INCRBYFLOAT
+- INFO
```

Comment:
@jhutchison We talked having an '*' next to this one and identifying what INFO information is supported.

Reply:
I can add-  is there an agreed upon existing verbage? or should I just write one?

Reply:
and since we are already using a superscript for slowlog, I would assume another superscript for INFO (as opposed to asterick)?